bitnami/wavefront/templates/NOTES.txt

```diff
@@ -0,0 +1,55 @@
+{{- $validClusterName := and (.Values.clusterName) (ne .Values.clusterName "KUBERNETES_CLUSTER_NAME") -}}
+{{- $validUrl := and (.Values.wavefront.url) (ne .Values.wavefront.url "https://YOUR_CLUSTER.wavefront.com") -}}
+{{- $validToken := and (.Values.wavefront.token) (ne .Values.wavefront.token "YOUR_API_TOKEN") -}}
+
+{{- if not $validClusterName }}
+
+##### ERROR:
+You must set the value for 'clusterName' to uniquely identify this Kubernetes cluster in Wavefront.
+#####
+{{- end }}
+
+{{- if .Values.collector.useProxy }}
+	{{- if and (not .Values.proxy.enabled) (not .Values.collector.proxyAddress) }}
+
+##### ERROR:
+Collector is set to use proxy but `proxy.enabled` is not true and `collector.proxyAddress` is not provided.
+#####
+	{{- end }}
+{{- else }}
+	{{- if or (not $validUrl) (not $validToken) }}
+
+##### ERROR:
+Collector is set to use direct ingestion API but `wavefront.url` or `wavefront.token` are not specified.
+#####
+	{{- end }}
+{{- end }}
+
+{{- if .Values.proxy.enabled }}
+	{{- if or (not $validUrl) (not $validToken) }}
+
+##### ERROR:
+Proxy is enabled but `wavefront.url` or `wavefront.token` are not specified.
+#####
+	{{- end }}
+{{- end }}
```

Comment:
We should do this with some `wavefront.validateValues` function (as we do in other charts)